sdk/formrecognizer/azure-ai-formrecognizer/README.md

```diff
@@ -586,42 +587,37 @@ with open(path_to_sample_documents, "rb") as f:
 result = poller.result()
 
 for idx, document in enumerate(result.documents):
-    print("--------Analyzing document #{}--------".format(idx + 1))
-    print("Document has type {}".format(document.doc_type))
-    print("Document has confidence {}".format(document.confidence))
-    print("Document was analyzed by model with ID {}".format(result.model_id))
+    print(f"--------Analyzing document #{idx + 1}--------")
+    print(f"Document has type {document.doc_type}")
+    print(f"Document has document type confidence {document.confidence}")
```

Comment:
```suggestion
    print(f"Document has confidence {document.confidence}")
```

Reply:
This needs to be fixed in the samples. Let's fix it in separate PR.